### ISSUE TITLE
Accuracy and visualization of the Fine-tuning model

### ISSUE THREAD

**Issue by glopes00** [facebookresearch/mae#129]
First of all, I would like to thank you guys for your excellent work and for making the code available.

I had 2 main issues while fine-tuning the model:

1. When I fine-tune a model (with a dataset that I have, just with thousands of .png images) I always get a Acc@1 and Acc@5 of 100.0%. That does not look like realistic values and I cannot figure out what is the problem.
2. I tried to run your demo jupyter notebook to see the checkpoints .pth files that the Fine-tuning of the model created, and I always run to a _IncompatibleKeys error (this error doesn't occur when I ran pre-train .pth files).

I hope you guys can help me with this.
